src/v0/destinations/tiktok_ads/transform.js

```diff
@@ -129,12 +129,10 @@ const getTrackResponse = (message, Config, event) => {
 
 const trackResponseBuilder = async (message, { Config }) => {
   const { eventsToStandard, sendCustomEvents } = Config;
-
-  let event = message.event?.toLowerCase().trim();
-  if (!event) {
-    throw new InstrumentationError('Event name is required');
+  if (!message.event || typeof message.event !== 'string') {
+    throw new InstrumentationError('Either event name is not present or it is not a string');
```

Comment:
Can't we ourselves convert the event into string rather throwing an error ?

Reply:
Also can you try using [this](https://github.com/rudderlabs/rudder-transformer/blob/3e8790ae6e2cc2c33e9a2da21ec8c10a68b4219f/src/v0/util/index.js#L2084) global util function with the same methodology

Reply:
We can use the util but I think we shouldn't modify data being sent by the customer.

Reply:
We shouldn't update the customer's data until it's obvious that the customer is sending "true" instead of a boolean.  To remove bugsnag alert we are handling this. Now the customer has a visibility of the failure and they can fix that if they want.